packages/browser-integration-tests/suites/tracing/browsertracing/backgroundtab-pageload/test.ts

```diff
@@ -4,17 +4,24 @@ import type { Event } from '@sentry/types';
 import { sentryTest } from '../../../../utils/fixtures';
 import { getFirstSentryEnvelopeRequest } from '../../../../utils/helpers';
 
-sentryTest('should finish pageload transaction when the page goes background', async ({ getLocalTestPath, page }) => {
-  const url = await getLocalTestPath({ testDir: __dirname });
+sentryTest(
+  'should finish pageload transaction when the page goes background',
+  async ({ browserName, getLocalTestPath, page }) => {
+    // TODO: This is flakey on firefox... trace.status is sometimes undefined
```

Comment:
I think we don't even need the TODO here. IMO it's unlikely that we'll fix this any time soon.